pkg/virt-launcher/virtwrap/converter/arch/converter_test.go

```diff
@@ -14,7 +14,7 @@
  *
  */
 
-package converter
+package arch
```

Comment:
This is most likely out of scope for this PR, but I wonder what other tests could reside under it.